tests/integration/_experimental/object_detection/test_evaluator_multiclass_f1_optimal.py

```diff
@@ -33,31 +35,29 @@
 from tests.integration._experimental.object_detection.test_evaluator_multiclass_fixed import TEST_DATA
 from tests.integration.helper import fake_locator
 
+kolena.initialize(os.environ["KOLENA_TOKEN"], verbose=True)
```

Comment:
Why is this needed? Supposedly the `with_init` fixture, which has `autouse=True`, would have configured this.